packages/utilities/package.json

```
@@ -33,20 +33,19 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./src/index.js"
+      "import": "./src/index.ts"
     },
     "./hooks": {
-      "import": "./src/hooks/index.js"
+      "import": "./src/hooks/index.ts"
     }
   },
-  "main": "./src/index.js",
+  "main": "./src/index.ts",
   "files": [
     "src"
   ],
   "scripts": {
     "build": "echo 'This package does not have a build process.'",
```

What do you think about adding this check to the build option so when we run a build at the project level, it's a one command check that everything is in order?

```suggestion
    "build": "npm run check",
```

---

I'm good with this. I'll add the same change to the design system to this PR as well.

---

Actually, I think that will create conflicts with your PR. I'll leave it for now.